index.js

```diff
@@ -1,5 +1,78 @@
-console.log("hola");
 
+"use strict";
 const mainContainer = document.querySelector("#app");
+const listOfCards = document.querySelector("#lista");
+const shuffle = document.querySelector(".shuffle");
+const topCard = document.querySelector("#top-card");
+
+const cardsArr = [];
+const clubs = ["clubs", "diamonds", "hearts", "spades"];
+const cards = ["As", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K"];
```

Comment:
It is not safe to keep and modify global state some it is best to move all the card generation logic into it's own function like `generateCards` which returns me an array of generated cards without needing to keep track or access global state